actuator/robstride/src/supervisor.rs

```diff
@@ -556,39 +556,20 @@ impl Supervisor {
                                 } else {
                                     record.state.last_command = now;
                                 }
-                            } else {
-                                if let Err(e) = record.actuator.get_feedback().await {
-                                    error!("Failed to get feedback from actuator {}: {}", id, e);
-                                }
-                            }
+                            } 
                         } else {
                             warn!(
                                 "Actuator {} is not ready, {:?}",
                                 id, record.state.control_command
                             );
                         }
-                    } else {
-                        if let Err(e) = record.actuator.get_feedback().await {
-                            error!("Failed to get feedback from actuator {}: {}", id, e);
-                        }
                     }
                 }
 
                 // Check timeouts and print stats with write lock
                 drop(actuators_snapshot); // Drop read lock before taking write lock
                 let mut actuators = self.actuators.write().await;
 
-                for (&id, record) in actuators.iter_mut() {
-                    if record.state.last_feedback.elapsed()? > self.feedback_timeout {
-                        error!("Feedback timeout for actuator {}", id);
-                        record.state.enabled = false;
-
-                        if let Err(e) = record.actuator.disable(false).await {
-                            error!("Failed to disable actuator {} after timeout: {}", id, e);
-                        }
-                    }
-                }
-
```

Comment:
actually i don't think we should delete this.

we just want to add functions to be able to use the supervisor without calling the `run` method